EasyPost.Net/Client.cs

```diff
@@ -34,59 +34,78 @@ public int RequestTimeoutMilliseconds
             set => _requestTimeoutMilliseconds = value;
         }
 
-        private string UserAgent => $"EasyPost/v2 CSharpClient/{_libraryVersion} .NET/{_dotNetVersion}";
+        private string UserAgent => $"EasyPost.Net/v2;CSharpClient/{_libraryVersion};.NET/{_dotNetVersion}";
 
         /// <summary>
-        ///     Constructor for the EasyPost client.
+        ///     Constructor for the EasyPost.Net client.
         /// </summary>
         /// <param name="clientConfiguration">EasyPost.ClientConfiguration object instance to use to configure this client.</param>
         public Client(ClientConfiguration clientConfiguration)
         {
             ServicePointManager.SecurityProtocol |= Security.GetProtocol();
-            _configuration = clientConfiguration ?? throw new ArgumentNullException("clientConfiguration");
-
-            _restClient = new RestClient(clientConfiguration.ApiBase);
-            _restClient.Timeout = ConnectTimeoutMilliseconds;
+            _configuration = clientConfiguration ?? throw new ArgumentNullException(nameof(clientConfiguration));
 
             Assembly assembly = Assembly.GetExecutingAssembly();
             FileVersionInfo info = FileVersionInfo.GetVersionInfo(assembly.Location);
             _libraryVersion = info.FileVersion;
 
             _dotNetVersion = Environment.Version.ToString();
+
+            RestClientOptions clientOptions = new RestClientOptions()
+            {
+                Timeout = ConnectTimeoutMilliseconds,
+                BaseUrl = new Uri(clientConfiguration.ApiBase)
+            };
+
+            _restClient = new RestClient(clientOptions);
         }
 
         /// <summary>
-        ///     Execute a request against the EasyPost API.
+        ///     Execute a request against the EasyPost.Net API.
```

Comment:
All these `.Net` instances need to be removed in comments please throughout the whole project. I assume they are leftover from branching off the last PR?

Reply:
Resolved in [d903ec7](https://github.com/EasyPost/easypost-csharp/pull/214/commits/d903ec7476be02f732e6eb4245a85595fc72df59)